lib/still/compiler/file.ex

```diff
@@ -28,4 +28,35 @@ defmodule Still.Compiler.File do
         msg
     end
   end
+
+  defp run_preprocessor(file) do
```

Comment:
What are your thoughts on doing this [in here](https://github.com/still-ex/still/blob/master/lib/still/preprocessor.ex#L109)?

Reply:
Yeah, much better, really 👍

Reply:
@frm you marked it as resolved but you didn't change this file right?

Reply:
Wasn't committed for some reason.